doc/zhCN/user/source/data/webadmin/index.rst

```diff
@@ -0,0 +1,26 @@
+.. _data_webadmin:
+
+数据设置
+=============
+
+本节介绍如何在GeoServer Web界面中管理加载，管理和发布数据。
+
+它描述了GeoServer用于访问和发布地理空间信息的核心配置数据类型。每个小节都介绍了提供添加，查看，编辑和删除功能的数据类型页面。
+
+.. toctree::
+   :maxdepth: 2
+
+   layerpreview
+   workspaces
+   stores
+   layers
+   layergroups
+
+.. note:: 在 ref:`styling_webadmin` 页面上可以找到样式信息。 
+
+
+
+
+
+
+

```

Comment:
Unnecessary trailing blank lines (or whitespaces) should be removed.

If you are using the VSCode editor, the following settings will trim those things for you:
![image](https://user-images.githubusercontent.com/64795/129126003-29391cdb-24b4-4508-aa31-42e7a69682fc.png)